src/sentry/static/sentry/app/components/charts/percentageTableChart.jsx

```diff
@@ -191,4 +198,20 @@ const CountColumn = styled(Name)`
   margin-left: ${space(0.5)};
 `;
 
-export default StyledPercentageTableChart;
+const TableHeader = styled(PanelItem)`
+  color: ${p => p.theme.gray2};
+  padding: ${space(1)};
+`;
+
+const TableChartWrapper = styled('div')`
+  margin-bottom: 0;
+  width: 100%;
+  padding: 0 ${space(2)};
+
+  /* stylelint-disable-next-line no-duplicate-selectors */
```

Comment:
not sure I understand the purpose of this block...

Reply:
stylelint yells when we use component as a selector (and we're using it because it's slightly complicated to style otherwise).